sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -167,15 +167,6 @@ abstract class ExternalCatalogSuite extends SparkFunSuite with BeforeAndAfterEac
     assert(actual.tableType === CatalogTableType.EXTERNAL)
   }
 
-  test("create table when the table already exists") {
```

Comment:
We check table existence in `SessionCatalog` instead of `ExternalCatalog` now. 
`SessionCatalogSuite` has similar test case, so no need to create new one.